### DISCUSSION TITLE
Twurple integration

### DISCUSSION THREAD

**Discussion by 111100001** [MrBrax/LiveStreamDVR#443] [General]
would it be better to use Twurple instead of creating a reverse proxy to run this application or not??

**Comment by 111100001** [comment]
https://github.com/twurple/twurple

**Comment by MrBrax** [comment]
it seems that this library is for querying twitch which i already do with own code, not sure if this would work better. i don't think it has anything to do with a reverse proxy

**Comment by 111100001** [comment]
maybe this could be helpful 
https://twurple.js.org/docs/getting-data/eventsub/reverse-proxy.html

**Reply by MrBrax** [reply]
that's basically what exists already, it's just configuration to allow a reverse proxy

**Reply by 111100001** [reply]
oh sorry then. im still learning all of this and hope one day i can get everything working fine
